python/perspective/perspective/client/table_api.py

```diff
@@ -0,0 +1,147 @@
+################################################################################
+#
+# Copyright (c) 2019, the Perspective Authors.
+#
+# This file is part of the Perspective library, distributed under the terms of
+# the Apache License 2.0.  The full license can be found in the LICENSE file.
+#
+from random import random
+from functools import partial
+from .dispatch import async_queue, subscribe, unsubscribe
+from .view_api import view as make_view
+
+
+def table(client, data, index=None, limit=None, name=str(random())):
+    """Create a Perspective `Table` by posting a message to a Perspective
+    server implementation through `client`, returning a `PerspectiveTableProxy`
+    object whose API is entirely async and must be called with `await` or
+    in a `yield`-based generator."""
+    options = {}
+
+    if index:
+        options["index"] = index
+    elif limit:
+        options["limit"] = limit
+
+    msg = {"cmd": "table", "name": name, "args": [data], "options": options}
+
+    proxy = PerspectiveTableProxy(client, name)
+    client.post(msg)
+    return proxy
+
+
+class PerspectiveTableProxy(object):
+    def __init__(self, client, name):
+        """A proxy for a Perspective `Table` object elsewhere, i.e. on a remote
+        server accessible through a Websocket.
+
+        All public API methods on this proxy are async, and must be called
+        with `await` or a `yield`-based coroutine.
+
+        Args:
+            client (:obj:`PerspectiveClient`): A `PerspectiveClient` that is
+                set up to send messages to a Perspective server implementation
+                elsewhere.
+
+            name (:obj:`str`): a `str` name for the Table. Automatically
+                generated if using the `table` function defined above.
+        """
+        self._client = client
+        self._name = name
+        self._async_queue = partial(async_queue, self._client, self._name)
+        self._subscribe = partial(subscribe, self._client, self._name)
+        self._unsubscribe = partial(unsubscribe, self._client, self._name)
+
+    def make_port(self):
+        return self._async_queue("make_port", "table_method")
+
+    def remove_port(self):
+        return self._async_queue("remove_port", "table_method")
+
+    def compute(self):
```

Comment:
Unrelated - but I think we can deprecate this "feature flag" - its implementation returns true for JS and Python